scripts/install_dependencies.sh

```diff
@@ -32,17 +32,19 @@ if [[ -z "${_ROCM_DIR}" ]]; then
 fi
 
 # Location of dependencies source code
-export _INSTALL_DIR=$BUILD_DIR/install
-export _DEPS_SRC_DIR=$_INSTALL_DIR/src
+_BUILD_DIR=${BUILD_DIR:-$PWD}
+export _INSTALL_DIR=${INSTALL_DIR:-$_BUILD_DIR/install}
+echo "rocSHMEM dependencies UCX and Open MPI will install in $_INSTALL_DIR"
 
+export _DEPS_SRC_DIR=$_BUILD_DIR/deps-src
 mkdir -p $_DEPS_SRC_DIR
 
 #Adjust branches and installation location as necessary
-export _UCX_INSTALL_DIR=$_INSTALL_DIR/ucx
+export _UCX_INSTALL_DIR=${INSTALL_DIR:-$_INSTALL_DIR/ucx}
```

Comment:
Maybe have a separate `UCX_INSTALL_DIR` and `OMPI_INSTALL_DIR` env vars that can be set, defaulting to `${_INSTALL_DIR}/ucx` (and `${_INSTALL_DIR}/ompi` if unset? Since `_INSTALL_DIR` is set to `${INSTALL_DIR}` if that exists anyway. Users do not necessarily want those installed in the same directory when `INSTALL_DIR` is set.
```suggestion
export _UCX_INSTALL_DIR=${UCX_INSTALL_DIR:-${_INSTALL_DIR}/ucx}
```

Reply:
I did that change because some users forgot to set LD_LIBARY_PATH and would get a different ucx than what ompi is built with. Installing them in the same directory helps removing that user-error since mpiexec prefix_by_default will include the proper LD_LIBRARY_PATH for both libmpi and libucx.